scanner/src/main/kotlin/storages/FileBasedStorage.kt

```diff
@@ -1,6 +1,5 @@
 /*
  * Copyright (C) 2017-2019 HERE Europe B.V.
- * Copyright (C) 2019 Bosch Software Innovations GmbH
```

Comment:
I guess this Copyrigth statement was removed accidentally.

Reply:
Yes, copy and paste mistake.